ci/build_python.sh

```diff
@@ -13,14 +13,9 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
-package_name="rmm"
-package_dir="python/rmm"
-
 version=$(rapids-generate-version)
```

Comment:
Out of scope for this PR, but a small idea for improvement: we ought to change `rapids-generate-version` to something like `rapids-overwrite-version <path_to_file>` that also does the echo overwriting. Then these build scripts become nearly trivial.

Even more out of scope for this PR: I would probably recommend tackling https://github.com/rapidsai/build-planning/issues/49 before making any further changes to gha-tools.

Reply:
We could get that behavior (cutting out an assignment here) with 0 changes to `rapids-generate-version`.

This:

```bash
version=$(rapids-generate-version)
echo "${version}" > VERSION
# ...
RAPIDS_PACKAGE_VERSION=${version} rapids-conda-retry mambabuild -c "${CPP_CHANNEL}" conda/recipes/rmm
```

Could become this:

```bash
rapids-generate-version > VERSION
# ...
RAPIDS_PACKAGE_VERSION=${head -1 VERSION} rapids-conda-retry mambabuild -c "${CPP_CHANNEL}" conda/recipes/rmm
```

I'm gonna need to push another commit here anyway to add the alpha spec and maybe the config setting for `disable-cuda` (once https://github.com/rapidsai/rapids-build-backend/pull/37 is released). Want me to make that change too?

Reply:
Yeah sure, why not. I'm all for brevity. Let's port that to all the repos while we're at it (CC @KyleFromNVIDIA).

Reply:
Alright thanks, I just pushed commits here doing that for all uses of `rapids-generate-version` in these build scripts.

Reply:
I've done the same in my PRs.